lib/storage/src/content_manager/consensus_manager.rs

```diff
@@ -703,6 +717,22 @@ impl<C: CollectionContainer> ConsensusManager<C> {
     }
 
     pub fn sync_local_state(&self) -> Result<(), StorageError> {
+        // Update our own metadata if outdated
+        if self.persistent.read().is_our_metadata_outdated() {
+            log::debug!("Proposing consensus peer metadata update for this peer");
+            let result = self
+                .propose_sender
+                .send(ConsensusOperations::UpdatePeerMetadata {
+                    peer_id: self.this_peer_id(),
+                    metadata: PeerMetadata::current(),
+                });
+            if let Err(err) = result {
+                log::error!(
+                    "Failed to propose consensus peer metadata update for this peer: {err}"
+                );
+            }
+        }
```

Comment:
For compatibility reasons we must disable this logic before merging, until we release 1.9.

Having this here in 1.8 will break clusters that are still running 1.7 nodes, so we need a migration period of at least 1 minor version.

Reply:
I've commented it out now so it is mergeable.